spec/system/admin/barcode_items_system_spec.rb

```diff
@@ -23,16 +23,20 @@
     end
 
     it "should edit an existing global barcode"
+
     it "should delete a global barcode" do
       visit admin_barcode_items_path
       page.refresh
-      expect(page).to have_content barcode_item.base_item.name
+      have_barcode_in_table = have_css(".td_barcode_name", exact_text: barcode_item.base_item.name)
+      expect(page).to have_barcode_in_table
+
       expect(
         accept_confirm do
           click_on "Delete"
         end
       ).to include "Are you sure you want to delete"
-      expect(page).to have_no_content barcode_item.base_item.name
+
+      expect(page).not_to have_barcode_in_table
     end
```

Comment:
`have_css` returns after finding the element on the page OR the default wait time is reached after the full page load. This means chaining it with not_to here will cause the test to wait unnecessarily

I think we can just make a call with `have_no_css` to remedy this problem..